src/suites/rulesets/lighthouse-api-standards.yaml

```
@@ -2,6 +2,16 @@ extends: 'spectral:oas'
 rules:
   operation-tag-defined: off
 
+  va-property-names-booleans:
+    description: Booleans should be prefixed with an auxiliary verb (such as is, has, or can).
+    message: '`{{property}}` is not prefixed with an auxiliary verb'
```

Do you think we should include an examples of an auxiliary verb?
```suggestion
    message: '`{{property}}` is not prefixed with an auxiliary verb (e.g. "veteran" becomes "isVeteran")'
```